### PR TITLE
Netcdf

### PR DESCRIPTION
**Pull request by JuliaKukulies**
* [ -] Have you added unit tests for the new functionality?
* [ x] Have all tests passed in your local clone?
* [ x] Have you written documentation that is easy to understand?
* [ x] Have you added an example?
* [ x] Has the formatting of your code been checked?

### CODE REVIEW

File: test/unit_tests/download/test_providers.py [hunk 1]
```diff
@@ -19,24 +19,3 @@ def setup_cds_identity(monkeypatch):
 
 
 HAS_PANSAT_PASSWORD = "PANSAT_PASSWORD" in os.environ
-
-
-@pytest.mark.skipif(not HAS_PANSAT_PASSWORD, reason="Pansat password not set.")
-@pytest.mark.usefixtures("test_identities")
-def test_copernicus_provider(tmpdir):
-    """
-    This test creates an instance for CopernicusProvider class and downloads global air temperatures for a randomly selected data product among available ERA5 reanalysis datasets.
```
Comment (simonpf) [inline test/unit_tests/download/test_providers.py:28]: This line is very looooooooooooooooooooooooooong.

**Review by simonpf — CHANGES_REQUESTED** [review]
xarray should be added to the dependencies as well.

**Comment by JuliaKukulies** [comment]
> xarray should be added to the dependencies as well.

fixed.

File: test/unit_tests/download/test_providers.py (hunk 1 above)
Comment (JuliaKukulies) [reply test/unit_tests/download/test_providers.py:28]: This test function is not there anymooooore!

**Pull request merged**